x-pack/plugins/canvas/public/lib/fullscreen.js

```diff
@@ -5,21 +5,22 @@
  * 2.0.
  */
 
-import { platformService } from '../services';
+import { pluginServices } from '../services';
```

Comment:
I _think_ that this file is not used anymore. It might should have been removed as part of the switch to ReactRouter.  I'm going to double check and remove it if so.

Reply:
Rock on.